### PR TITLE
docs: update entitymanager definition

### PR DESCRIPTION
**Pull request by Parsath**
### Description of change

change the "What is EntityManager?" page to be up-to-date with v 0.3.0

1. line 6 changes from
`You can access the entity manager via 'getManager()' or from 'Connection'.`
to 
`You can access the entity manager via DataSource's manager.`

2. the import on `getManager` in line 10 becomes `DataSource`  directly imported from typeorm:
`import {getManager} from "typeorm";`
becomes 
`import {DataSource} from "typeorm";`

3. change entityManager definition in line 13 and user load in line 14:
from
`const entityManager = getManager(); // you can also get it via getConnection().manager`
`const user = await entityManager.findOne(User, 1);`
to
`const myDataSource = new DataSource({ /*...*/ });`
`const user = await myDataSource.manager.findOne(User, 1);`

### Pull-Request Checklist

- [ ] Code is up-to-date with the `master` branch
- [ ] `npm run lint` passes with this change
- [ ] `npm run test` passes with this change
- [ ] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: docs/working-with-entity-manager.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 Using `EntityManager` you can manage (insert, update, delete, load, etc.) any entity. 
 EntityManager is just like a collection of all entity repositories in a single place.
  
-You can access the entity manager via `getManager()` or from `Connection`.
+You can access the entity manager via DataSource's manager.
 Example how to use it:
  
 ```typescript
-import {getManager} from "typeorm";
+import {Manager} from "./config/DataSource";
```
Comment (pleerock) [inline docs/working-with-entity-manager.md:10]: it's unclear what `./config/DataSource` is. To keep it simple I think we should simply show how to intiailize a data source and access it's manager, e.g.:

```ts
const myDataSource = new DataSource({ /*...*/ });
const user = await myDataSource.manager.findOne(User, 1);
user.name = "Umed";
await entityManager.save(user);
```

**Comment by pleerock** [comment]
thanks for contribution and btw set PR base to `0.3.0` branch instead of `master` branch. Thanks!

File: docs/working-with-entity-manager.md (hunk 1 above)
Comment (Parsath) [reply docs/working-with-entity-manager.md:10]: Thank you for your feedback!

**Comment by pleerock** [comment]
Thank you again!

**Pull request merged**